create_plots.py

```diff
@@ -0,0 +1,24 @@
+import argparse
```

Comment:
that's starting to be a lot of files with a single function. try to put things otgether. 

For example, we already have a whole `viz` directory (which honestly we could put in utils=

why is this cli not with `run_experiments.py` ?